Composer/plugins/azurePublish/src/index.ts

```diff
@@ -195,11 +195,16 @@ class AzurePublisher {
       }
     } catch (error) {
       console.log(error);
+      if (typeof error === 'object') {
```

Comment:
I'd be happier if we could guarantee we'd see actual `Error` objects whenever an exception is thrown (because those have a known shape instead of just being generic objects, and then we could just test `error.message`), but I don't know how we can get that assurance. MDN says that literally anything can be used with `throw`, so it looks like we do need a check like this.

However, I think we should first check if `error` is an instance of `Error`, because then if someone is nice enough to follow the conventions, we can then log `error.message` instead of spitting out raw JSON.

Reply:
done